### ISSUE TITLE
Using query builder "select" method results in a weird false positive

### ISSUE THREAD

**Issue by hailwood** [nunomaduro/larastan#172]
- Larastan Version: 0.3.4
- `--level` used: 5 (default)

### Description:
Using the query builder `select` method seems to set the execution context to the current class e.g.

If I am inside a class `class MyClass {}` and call 
```php
Product::select(['code', 'quantity_in_stock'])->where(['status' => true])->get()
```
then I get the false positive of 

> Call to undefined method App\MyClass::get()

Updating the code to remove the select I.e. 
```php
Product::where(['status' => true])->get()
```
Does not produce the false positive (of course it's no longer doing what I want...)

**Comment by alexey-m-ukolov** [comment]
I think it's the same problem as in #168

**Comment by nunomaduro** [comment]
@hailwood @alexey-m-ukolov This issue has been fixed in Larastan `v0.3.10`. 

Please run a `composer update nunomaduro/larastan` and tell me the results.
